templates/taxi/car_list.html

```diff
@@ -7,15 +8,44 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="🔍" class="btn btn-secondary">
+  </form>
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+            <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
There is an extra space before the closing quotation mark in the URL tag. It should be `{% url 'taxi:car-detail' pk=car.id %}` without the space.